src/wasm.cc

```diff
@@ -121,60 +121,22 @@ void WasmBase::registerCallbacks() {
   _REGISTER_WASI(proc_exit);
```

Comment:
WASI calls should be also covered.

Reply:
Fixed, and added/updated tests in the Envoy PR.